src/utils.coffee

```diff
@@ -18,7 +18,7 @@ device = require './device'
 exports.supervisorVersion = require('./lib/supervisor-version')
 
 configJson = require('/boot/config.json')
-if Boolean(configJson.supervisorOfflineMode)
+if Boolean(config.apiEndpoint) and !Boolean(configJson.supervisorOfflineMode)
```

Comment:
The thing is the config.coffee file doesn't currently use config.json and I don't want to mix it up in this PR as it's gonna be messy. I'm working on a better fix (consolidate *all* configuration into a single module) but I want to get this in before (since we need it rather urgently).